prime-router/src/test/kotlin/azure/HttpTestUtils.kt

```diff
@@ -15,13 +15,14 @@ import kotlin.collections.Map
 class MockHttpResponseMessage : HttpResponseMessage.Builder, HttpResponseMessage {
     var httpStatus: HttpStatusType = HttpStatus.OK
     var content: Any? = null
+    var headers: MutableMap<String, String> = mutableMapOf()
 
     override fun getStatus(): HttpStatusType {
         return this.httpStatus
     }
 
     override fun getHeader(var1: String): String {
-        return "world"
```

Comment:
Not sure what the thought was here, but this needed to change in order to test that the location was returned correctly